client/src/admin/NotificationsPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NotificationsPage = () => (
+  <div className="container--content">
```

Comment:
While this does technically add a notifications page, why do we not have anything actually displaying for notifications? Even though the end point doesn't exist yet I'm hesitant to merge this as a dummy page.